scripts/fast-reboot-dump.py

```diff
@@ -49,24 +53,56 @@ def get_vlan_ifaces():
 
     return vlans
 
+def get_bridge_port_id_2_port_id(db):
+    bridge_port_id_2_port_id = {}
+    keys = db.keys(db.ASIC_DB, 'ASIC_STATE:SAI_OBJECT_TYPE_BRIDGE_PORT:oid:*')
```

Comment:
I run this command before the switch goes into the FR. I think external tools should check a state of a switch before the initiate FR.